iceoryx_posh/test/moduletests/test_capro_service.cpp

```diff
@@ -365,146 +267,120 @@ TEST_F(ServiceDescription_test, ServiceDescriptionCtorWithServiceStringAndInstan
 
 TEST_F(ServiceDescription_test, TwoServiceDescriptionsWithAnyServiceAnyInstanceAnyEventIDsAreEqual)
 {
-    uint16_t testServiceID = iox::capro::AnyService;
-    uint16_t testEventID = iox::capro::AnyEvent;
-    uint16_t testInstanceID = iox::capro::AnyInstance;
-    ServiceDescription serviceDescription1 = ServiceDescription(testServiceID, testEventID, testInstanceID);
-    ServiceDescription serviceDescription2 = ServiceDescription(testServiceID, testEventID, testInstanceID);
+    IdString_t testService = iox::capro::AnyServiceString;
```

Comment:
Just a personal preference, but I found `testServiceID` a bit less ambiguous

Reply:
Considering it is not even really an ID in the strict sense (it is a partial key to identify the service)  it does not matter and shorter is better here.